src/test/java/core/basesyntax/service/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static BalanceOperationHandler balanceOperationHandler;
+
+    @Before
+    public void setUp() {
+        balanceOperationHandler = new BalanceOperationHandler();
+    }
+
+    @After
+    public void afterEachTest() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    public void balanceOperation_putBalanceToMap_ok() {
```

Comment:
```suggestion
    public void handle_ok() {
```
![image](https://user-images.githubusercontent.com/20326858/229105188-2cc9fac0-7b22-4315-aa8b-03462a1f9755.png)